script/StorageRent.s.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.18;
```

Comment:
thoughts on upgrading all our contracts to 8.20?

Reply:
Depends if our target L2 supports `PUSH0`. I don't think Optimism/Arbitrum do yet, but I'll double check.

Reply:
oh you're right, they dont